src/client/websocket/handlers/GUILD_MEMBERS_CHUNK.js

```diff
@@ -17,6 +17,14 @@ module.exports = (client, { d: data }) => {
    * @event Client#guildMembersChunk
    * @param {Collection<Snowflake, GuildMember>} members The members in the chunk
    * @param {Guild} guild The guild related to the member chunk
+   * @param {object} chunk Properties of the received chunk
```

Comment:
So that the external MDN source gets linked on the docs website
```suggestion
   * @param {Object} chunk Properties of the received chunk
```

Reply:
👌